screenpipe-server/src/bin/screenpipe-server.rs

```diff
@@ -174,7 +174,12 @@ async fn main() -> anyhow::Result<()> {
                 }
 
                 info!("screenpipe setup complete");
-                // TODO: ffmpeg sidecar thing here
+                if let Some(ffmpeg_path) = find_ffmpeg_path() {
```

Comment:
we should download ffmpeg here using the lib, as this is used in app onboarding

Reply:
@louis030195 Look, it already does that: https://github.com/mediar-ai/screenpipe/blob/704762f52279fd4fbc1e7a2d52835ded18e96f46/screenpipe-core/src/ffmpeg.rs#L87